styles.css

```diff
@@ -496,3 +501,104 @@ h2 {
   align-items: center;
   justify-content: space-between;
 }
+
```

Comment:
- [ ] Please remove horizontal scrolling from your body to make it more professional and match the figma design.
![image](https://user-images.githubusercontent.com/91486791/220368747-7467281a-79d7-4293-9371-1941febdd129.png)